pytket/qir/conversion/api.py

```diff
@@ -221,7 +224,9 @@ def check_circuit(
 
     :param circuit: given circuit
     :param int_type: integer bit width (32 or 64)
-    :raises ValueError: with a suggestion on how to resolve the problems
+    :param gate_set: set of OpTypes to use to check that all gates can be converted
```

Comment:
Should mention default if omitted.

Reply:
Done in [35922b6](https://github.com/CQCL/pytket-qir/pull/229/commits/35922b6879a183f9ed219f3dc8f8f476937d676e)